Watchman.Discord/Areas/Reactions/Contexts/ReactionContext.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using Discord;
+using Discord.WebSocket;
+
+namespace Watchman.Discord.Areas.Reactions.Contexts
```

Comment:
czemu twierdzisz że powinniśmy to tworzyć osobno za każdym razem, kiedy tworzymy bota za pomocą tego frameworka?

Reply:
`Areas.Reactions.Contexts`
Contexty powinny być dostępne dla wszystkich użytkowników frameworka
a nie JEDYNIE w tym bocie
i JEDYNIE w jednym konkretnym miejscu